src/api/spec/features/webui/requests_spec.rb

```diff
@@ -374,4 +374,42 @@
       expect(page).not_to have_text('Project Maintainers')
     end
   end
+
+  describe 'a request submitted against a staging project' do
+    let!(:workflow) do
+      create(:staging_workflow_with_sources)
+      Staging::Workflow.last
+    end
+    let!(:staging_project) { workflow.staging_projects.first }
```

Comment:
```suggestion
    let(:staging_project) { workflow.staging_projects.first }
```
You don't really need the bang here, as you can just lazy load it when needed